### PR TITLE
LPD-39651 - Refactor LocalFile.CountriesManagement#CanActivateCountriesInBulk into Playwright

### PR DESCRIPTION
**Pull request by matyaswollner**
https://liferay.atlassian.net/browse/LPD-39651

### CODE REVIEW

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:sf

File: modules/test/playwright/pages/address-web/CountriesManagementPage.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+			exact: true,
+			name: 'Filter',
+		});
+		this.page = page;
+	}
+
+	async changeFilter(option: 'Active' | 'Inactive' | 'All') {
+		await clickAndExpectToBeVisible({
+			autoClick: true,
+			target: this.page.getByRole('menuitem', {
```
Comment (smottal) [inline modules/test/playwright/pages/address-web/CountriesManagementPage.ts:89]: cn you create a locator for this?

File: modules/test/playwright/pages/address-web/CountriesManagementPage.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+				exact: true,
+				name: option,
+			}),
+			trigger: this.filterButton,
+		});
+
+		await this.page
+			.getByText('Search Results', {exact: true})
```
Comment (smottal) [inline modules/test/playwright/pages/address-web/CountriesManagementPage.ts:97]: also for this

File: modules/test/playwright/tests/address-web/countriesManagement.spec.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {countriesManagementPageTest} from '../../fixtures/CountriesManagementPageTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {waitForAlert} from '../../utils/waitForAlert';
+
+export const test = mergeTests(loginTest(), countriesManagementPageTest);
+
+test('LPD-39651 - Can activate countries in bulk', async ({
```
Comment (smottal) [inline modules/test/playwright/tests/address-web/countriesManagement.spec.ts:14]: please remove the - in the name of the test

**Review by smottal — CHANGES_REQUESTED** [review]
@matyaswollner good job! Just very small changes

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by smottal** [comment]
@matyaswollner I resent it here: https://github.com/liferay-commerce/liferay-portal/pull/5534 adding a SF commit (I don't know if this is something related but know the SF CI passes)

**Pull request closed without merging**